README.md

```diff
@@ -27,7 +27,8 @@ Github Actions currently builds a Windows x64 `.dll`, A Linux x64 `.so`, and a m
 You need:
 * git
 * Dart 3+
-* C++ build tools for your platform (Visual Studio, XCode, gcc, etc)
+* C++ build tools for your platform (Visual Studio, XCode, gcc, etc) 
+* For Windows VS 2019 16.61 with 10.0.20348.0 SDK don't forget install Debugger Tools
```

Comment:
I think any version of Visual Studio 2019+ should work. I'm using 2022 I think....

Reply:
This is difficlut, I you have more than one IDE / Compiler on Machine

see dart_shared_libray\dart-sdk\sdk\build\vs_toolchain.py

That was my Build Problem at first time i get always an fiele not found from windows SDK.
wenn it found an 2019 it build with this and that it muss be 10.0.20348.0 as SDK
I don't know witch SDK version is used with 2022.
I want to say with 2019 it work only with this SDK

Reply:
Left a comment on the PR. The officially supported toolchains and what actually works are sometimes different :) I don't have 2019, I only have 2022, but if you look at `setupEnv.ps1` I tell the script it's Visual studio 2019.

The key I think is just VS2019 and the SDK version you mentioned.

Reply:
I don't use setupEnv.ps1 only the DEPOT_TOOLS_WIN_TOOLCHAIN=0 is an must.
I have 2019 and 2022 Prof on my Machine
Don't think it is good Idea say there ist an 2019 and Complie with 2022.
I think we need an better description in readme vor that.